src/linear_solver_logs/solve_log_ma.jl

```diff
@@ -75,14 +100,17 @@ A mutable structure that stores information about a randomized linear solver's b
 - `iterations::Int64`, the number of iterations of the solver.
 - `converged::Bool`, a flag to indicate whether the system has converged by some measure.
 - `true_res`, a boolean indicating if we want the true residual computed instead of approximate.
-- `dist_info::DistInfo`, a structure that stores information about the sub-Exponential distribution.
+- `dist_info::SEDistInfo`, a structure that stores information about the sub-Exponential distribution.
+  includes the parameter `sigma2`, which is the variance parameter of a sub-Exponential distribution,
+  and the parameter `omega`. This values have been approximated for many sampling methods; but the 
+  user can specify there own, if desired. The parameter `eta` can also be used to scale the default
+  parameters by having the variance be set to `sigma2/eta` and the other parameter be `omega/eta`.
+
 # Constructors
 - Calling `LSLogMA()` sets `collection_rate = 1`,  `lambda1 = 1`,
-    `lambda2 = 30`, `resid_hist = Float64[]`, `iota_hist = Float64[]`, `width_hist = Int64[]`, 
-    `resid_norm = norm` (Euclidean norm), `iterations = -1`, `converged = false`, 
-    `sampler = LinSysVecRowDetermCyclic`, `max_dimension = 0`, `sigma2 = nothing`, `omega = nothing`,
-    `eta = 1`, and `true_res = false`. The user can specify their own values of lambda1, lambda2, sigma2, omega, and eta using 
-    key word arguments as inputs into the constructor.
+    `lambda2 = 30`, `resid_norm = norm` (Euclidean norm), `sigma2 = nothing`, `omega = nothing`,
+    `eta = 1`, and `true_res = false`. The user can specify their own values of `collection_rate`,
+    `lambda1`, `lambda2`, `sigma2`, `omega`, and `eta` using key word arguments as inputs into the constructor.
```

Comment:
The doctstring for the constructor needs to follow the typical format of a function

Reply:
Because the variables were defined above I did not exactly follow the function format as it would be repetitive; however, I did rewrite the default values in a list so that it is hopefully more clear to look at.